component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/backupcode/BackupCodeAuthenticator.java

```diff
@@ -153,6 +159,14 @@ protected void initiateAuthenticationRequest(HttpServletRequest request, HttpSer
 
         String username = null;
         String tenantDomain = context.getTenantDomain();
+
+        Map<String, String> parameterMap = getAuthenticatorConfig().getParameterMap();
+        boolean showAuthFailureReason = Boolean.parseBoolean(parameterMap.get(FrameworkConstants.SHOW_AUTHFAILURE_RESON_CONFIG));
```

Comment:
check if this line is exceeding 120 characters

Reply:
Addressed in 4a5ea8c132363c3a1c4adec85f40b4605566a7e6